Makefile

```diff
@@ -38,6 +38,9 @@ BUNDLE_IMG ?= $(IMAGE_TAG_BASE)-bundle:v$(VERSION)
 # BUNDLE_GEN_FLAGS are the flags passed to the operator-sdk generate bundle command
 BUNDLE_GEN_FLAGS ?= -q --overwrite --version $(VERSION) $(BUNDLE_METADATA_OPTS)
 
+# tripleo-ansible-ee image
+EEIMG ?= quay.io/openstack-k8s-operators/tripleo-ansible-ee
```

Comment:
As per the changes merged just minutes ago, I propose to change this to quay.io/openstack-k8s-operators/openstack-ansibleee-runner. That would make it more in line with the rename process, will remove the tripleo reference, which is not needed anymore and will specify that this is the ansible runner. What do you think?